Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -201,6 +205,12 @@ private void Awake()
         {
             queryBufferNearObjectRadius = new SpherePointerQueryInfo(sceneQueryBufferSize, Mathf.Max(NearObjectRadius, SphereCastRadius), NearObjectSectorAngle, PullbackDistance, nearObjectSmoothingFactor, true);
             queryBufferInteractionRadius = new SpherePointerQueryInfo(sceneQueryBufferSize, SphereCastRadius, 360.0f, 0.0f, 0.0f);
+            nearInteractionGrabbableCache = new LRUCache<int, NearInteractionGrabbable>(nearInteractableCacheCapacity);
+        }
+
+        protected virtual void OnDestroy()
+        {
+            nearInteractionGrabbableCache.Clear();
```

Comment:
`nearInteractionGrabbableCache` looks like a non-static reference. Does it persist in some fashion when this component is entirely destroyed?

Reply:
I think you're correct, this is unneeded. I'll remove it unless @CharlesWanMS knows of some reason why this needs to be explicitly cleared.

Reply:
That is correct it's not strictly needed.